pkg/asset/templates/bootkube/constants.go

```diff
@@ -0,0 +1,23 @@
+package bootkube
+
+import (
+	"io/ioutil"
+
+	"github.com/openshift/installer/data"
+)
+
+const (
+	templateDir = "templates"
+	dataDir     = "manifests/bootkube/"
```

Comment:
All of your `getFileContents` calls currently include an explicit `path.Join(dataDir, ...)`.  Maybe just drop `dataDir` and push that `Join` down into `getFileContents`?

Reply:
Done.